looper/looper.py

```diff
@@ -44,13 +44,11 @@
 _LOGGER = logging.getLogger()
 
 
-def parse_arguments():
+def build_parser():
```

Comment:
Thanks for doing this @michalstolarczyk . This not only is useful for other packages' interaction with `looper`, but also could allow us to more thoroughly explore the full space of configurations and user options, and make assertions about their behavior. Nice work!